### DISCUSSION TITLE
ADd compatibility for fork of MPC-HC?

### DISCUSSION THREAD

**Discussion by Drudicta** [erengy/taiga#1170] [General]
I'm running a version of MPC-HC at https://github.com/clsid2/mpc-hc but it is not recognized by Taiga. Is there a way to add functionality to this fork so my anime can be recognized?

**Comment by erengy** [comment]
Just tried running the latest version (2.1.4 x64) from that link and Taiga was able to detect it as is.

If it's not being detected for you, first make sure you have MPC-HC enabled at *Settings → Recognition → Media players*. If that's not the issue, it might be because you're running MPC-HC as administrator.

**Comment by Drudicta** [comment]
I'll try a more common anime and try again, thank you.
